src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseChecklistServiceTests.cs

```diff
@@ -357,13 +375,17 @@ public async Task GetChecklist_FullyValid()
             var originalRelease = new Release
             {
                 Publication = publication,
-                Version = 0
+                Version = 0,
+                Created = DateTime.UtcNow.AddDays(-14),
             };
+            var releaseId = Guid.NewGuid();
```

Comment:
Is there any need to introduce this releaseId here? This release and it's notes will be assigned the release id when it's saved below. An explicit save of `release` could be added, although it should be saved when `releaseContentSection1` is saved.